_posts/2016-09-30-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: 2017 Kickstart locations announced
+---
+
+We are happy to announce the locations for our 2017 Kickstart.
+
+The venues for Student Robotics 2017 kickstart are:
+
+- Southampton University
+- Bristol University
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
```

Comment:
Not sure I follow this, would it be valid if I move this to a separate page and write a much more brief version as a news article?